enums/eventBrandMembership.enum.go

```diff
@@ -0,0 +1,23 @@
+package enums
+
+type EventMembershipRole int
+
+const (
+	Admin EventMembershipRole = iota
+	Contributor
+	Attendee
+)
+
+func GetRoleDescription(role int) string {
+	switch role {
+
+	case int(Admin):
+		return "admin"
+	case int(Contributor):
+		return "contributor"
+	case int(Attendee):
+		return "attendee"
+	default:
+		return "Invalid role"
+	}
```

Comment:
Better to throw an error or panic for default case rather than returning string value. As this case shouldn't exist.